controllers/hivelocitycluster_controller.go

```diff
@@ -127,7 +127,7 @@ func (r *HivelocityClusterReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	secretManager := secretutil.NewSecretManager(log, r.Client, r.APIReader)
 	apiKey, hvSecret, err := getAndValidateHivelocityAPIKey(ctx, req.Namespace, hvCluster, secretManager)
 	if err != nil {
-		return hvAPIKeyErrorResult(ctx, err, hvCluster, infrav1.HivelocityClusterReady, r.Client)
+		return hvAPIKeyErrorResult(ctx, err, hvCluster, infrav1.HivelocityClusterReadyCondition, r.Client)
```

Comment:
this should be CredentialsAvailableCondition no?

Reply:
right!

Reply:
then have to think where to make the condition true for the cluster

Reply:
https://github.com/hivelocity/cluster-api-provider-hivelocity/blob/c5be98c799432d14aba293b94fde3717a26d6d64/controllers/hivelocitycluster_controller.go#L374 This function we deal with a lot of secrets, do we need any credential available condition?

Reply:
just look in CAPH, where we have the same reconcileTargetSecret and reconcileTargetManager. This is not related to the other credentials.

Reply:
> then have to think where to make the condition true for the cluster

I am not sure whether we need it at all in the end.